.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: node_js
 node_js:
-  - "0.12"
+  - 6
+  - 4
```

Comment:
> At https://docs.travis-ci.com/user/languages/javascript-with-nodejs I find:

Interestingly, that link also seem to suggest that the version should be enclosed in quotes.